types/cronitor/index.d.ts

```diff
@@ -0,0 +1,115 @@
+    name?: string; // Display name of the monitor
+    note?: string; // Additional context/troubleshooting information
+    notify?: string[] | { alerts: string[]; events: Record<string, boolean> }; // Notification settings
+    paused?: boolean; // Monitoring/alerting status
+    platform?: string; // Platform information
+    grace_seconds?: number; // Number of seconds for grace period
+    tags?: string[]; // Array of tags
+    timezone?: string; // Timezone
+    schedule_tolerance?: number; // Schedule tolerance
+    failure_tolerance?: number; // Failure tolerance
+    created?: string; // Timestamp when the monitor was created
+    disabled?: boolean; // Current disabled status
+    initialized?: boolean; // Whether the monitor has received a telemetry event
+    passing?: boolean; // Whether the monitor is currently passing or failing
+    running?: boolean; // Whether a job is running (only applicable for type: job)
+    defaultName?: string; // Default name for provisioning
+    defaultNote?: string; // Default note for provisioning
+    realert_interval?: number | string; // Interval for follow-up alerts
+    request?: {
+        url: string;
+        method?: string;
+        body?: string;
+        headers?: Record<string, string>;
+        cookies?: Record<string, string>;
+        timeout_seconds?: number;
+        follow_redirects?: boolean;
+        verify_ssl?: boolean;
+    }; // required by type: check
+}
+
+declare class Monitor {
+    constructor(key: string);
+    get State(): MonitorState;
+    static put(data: MonitorAttributes | MonitorAttributes[], rollback?: boolean): Promise<Monitor | Monitor[]>;
+    data(): Promise<any>;
+    pause(hours: number): Promise<boolean>;
+    unpause(): Promise<boolean>;
+    ok(params?: TelemetryEvent): Promise<boolean>;
+    delete(): Promise<boolean>;
+    ping(params?: TelemetryEvent): Promise<boolean>;
+}
+
+declare class Event {
+    constructor(key: string, options?: { intervalSeconds?: number });
+    tick(count?: number): void;
+    error(count?: number): void;
+    stop(): Promise<void>;
+    fail(message?: string | null): Promise<void>;
+}
+
+declare class Cronitor {
+    constructor(apiKey: string, config?: CronitorConfig);
+    generateConfig(): Promise<void>;
+    applyConfig(rollback?: boolean): Promise<void>;
+    validateConfig(): Promise<void>;
+    readConfig(path: string | null, output?: boolean): void;
+    wrap(key: string, callback: () => Promise<any>): () => Promise<any>;
+    wraps(lib: any): void;
+    newSeries(): string;
+    schedule(key: string, schedule: string, cb: () => void, options?: any): any;
+
+    Monitor: typeof Monitor;
+    Event: typeof Event;
+
+    static State: {
+        RUN: "run";
+        COMPLETE: "complete";
+        FAIL: "fail";
+        OK: "ok";
+    };
+}
+
+declare function Cronitor(apiKey: string, config?: CronitorConfig): Cronitor;
+
+export = Cronitor;
+
+export as namespace Cronitor;
```

Comment:
```suggestion
```

It doesnt look like from documentation that the library supports using it as global scope. 
If it does please point to documentation.

Reply:
You're absolutely right. Fixed!